docs/content/CircleBadge.mdx

```diff
@@ -0,0 +1,93 @@
+---
+componentId: circle_badge
```

Comment:
Total nitpick, but why do we put `componentId` at the top? I feel like `title` should always go first and `componentId` at the bottom as it's the least useful for those scanning this file.

Reply:
I'm fine with either :) The only reason I put it on the top was to try to avoid merge conflicts with other PRs 😅